pkg/envtest/server.go

```diff
@@ -143,8 +149,28 @@ func (te *Environment) Start() (*rest.Config, error) {
 		}
 	} else {
 		te.ControlPlane = integration.ControlPlane{}
-		te.ControlPlane.APIServer = &integration.APIServer{Args: te.getAPIServerFlags()}
-		te.ControlPlane.Etcd = &integration.Etcd{}
+		if te.ControlPlane.APIServer == nil {
+			te.ControlPlane.APIServer = &integration.APIServer{Args: te.getAPIServerFlags()}
+		}
+		if te.ControlPlane.Etcd == nil {
+			te.ControlPlane.Etcd = &integration.Etcd{}
+		}
+
+		if os.Getenv(envAttachOutput) == "true" {
+			te.AttachControlPlaneOutput = true
+		}
+		if te.ControlPlane.APIServer.Out == nil && te.AttachControlPlaneOutput {
```

Comment:
we should make it easier to set these ourselves in case people want to redirect elsewhere.  Maybe we shouldn't clear `te.ControlPlane` above?

Reply:
> Maybe we shouldn't clear te.ControlPlane above?

Done. Now people can directly set `Out` and `Err` in `ControlPlane.APIServer` and `ControlPlane.Etcd`.